dev_local_env/files/amazon_linux_2/requirements.txt

```diff
@@ -0,0 +1,12 @@
+ansible==2.7.5    
```

Comment:
Thinking about this, these requirements will change depending on team. Like right now, we just upgraded to a higher version of Ansible and mitogen. Most teams are moving away from Ansible. I'm not sure we need anything for AL2 in terms of a dev environment since it would be covered by roles setting up our CI I think.